app/index.html

```diff
@@ -70,13 +70,10 @@
   <script src="_components/cbioportal-client.js"></script>
   <!-- endbuild -->
 
-  <!-- build:js scripts/vs-session.js -->
+  <!-- build:js scripts/iviz.js -->
   <script src="scripts/session/sessionEvent.js"></script>
```

Comment:
Are we no longer using session on other pages?

Reply:
for the frontend repo we are having its own typescript client